### DISCUSSION TITLE
There is a minor issue in the documentation

### DISCUSSION THREAD

**Discussion by AlephLau** [pimcore/pimcore#17119] [General]
This message could be confusing, especially when the same documentation tells you that 2023.3 is the LTS version but also tells you that this documentation is no longer maintained. I wanted to report it on the appropriate repository, but I think they no longer have the new Docusaurus documentation repository available to the public.

![image](https://github.com/pimcore/pimcore/assets/70108013/0fb78081-c945-4df5-864f-09f7dddd8556)

**Comment by fashxp** [comment]
thx for reporting, that is a valid point. We'll have a look...

**Reply by AlephLau** [reply]
You're welcome 😁
